packages/ToggleSwitch/ToggleSwitch.md

```diff
@@ -1,3 +1,46 @@
+### Usage Criteria
+
+- `ToggleSwitch` button is a controlled component that requires an `onClick` handler that will trigger `true` or `false` value of `aria-checked` attribute.
+- The main intent for a `ToggleSwitch` should be using it as a button. For example, you could trigger an API request or change some state of your application.
+- If the intent is to use it as a form input element with a designated value, please use either [@tds/core-checkbox](https://tds.telus.com/components/index.html#checkbox) or [@tds/core-radio](https://tds.telus.com/components/index.html#radio)
+- By default `ToggleSwitch` button sticks to the right side of the contrainer.
```

Comment:
**By default,** `ToggleSwitch` button sticks to the right side of the **container**

Reply:
Done